docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_tablespace.md

```diff
@@ -0,0 +1,41 @@
+---
+title: CREATE TABLESPACE [YSQL]
+headerTitle: CREATE TABLESPACE
+linkTitle: CREATE TABLESPACE
+description: Use the CREATE TABLESPACE statement to create a tablespace in the cluster.
+menu:
+  preview:
+    identifier: ddl_create_table
+    parent: statements
+type: docs
+---
+
+## Synopsis
+
+Use the `CREATE TABLESPACE` statement to create a tablespace in the cluster. It defines the tablespace name and tablespace properties.
+
+## Syntax
+
+{{%ebnf%}}
+  create_tablespace
+{{%/ebnf%}}
+
+## Semantics
+
+- Create a tablespace with *tablespace_name*. If `qualified_name` already exists in the cluster, an error will be raised.
+- YSQL tablespaces allow administrators to specify the number of replicas for a table or index, and how they can be distributed across a set of clouds, regions, and zones in a geo-distributed deployment.
+
+### *tablespace_option*
+
+- Can be one of [`replica_placement`].
+- Use `replica_placement` to specify the number of replicas stored in specific zones,regions or clouds.
```

Comment:
replica_placement, judging from the examples, has a specific syntax/content. Should we provide a quick example or description of how to specify placement here, rather than having to tease it out in the examples